neuroscout_cli/cli.py

```diff
@@ -22,8 +22,8 @@
     -e, --estimator <es>     Estimator to use for first-level model
                              [default: nistats]
     -f, --force-neurovault   Force upload, if a NV collection already exists
-    -d, --drop-missing       Drop missing contrast
-    -v, --verbose	     Verbose mode
+    -d, --drop-missing       Drop missing contrasts
+    -v, --verbose	         Verbose mode
```

Comment:
Tab character messing up visual alignment.

```suggestion
    -d, --drop-missing       Drop missing contrasts
    -v, --verbose            Verbose mode
```